lib/rester/client.rb

```diff
@@ -72,5 +79,15 @@ def _parse_json(data)
         {}
       end
     end
+
+    class Response < Hash
+      def initialize(status)
+        define_singleton_method(:status) { status }
+      end
+
```

Comment:
Why not `attr_reader :status` and then `@status = status`?

Also, I'm in favor of not exposing `status` unless necessary.  How about just `@status = status` and then access `@status` in `#successful?`?